python/pfs/drp/stella/datamodel/pfsFluxReference.py

```diff
@@ -0,0 +1,9 @@
+import pfs.datamodel.pfsFluxReference
+
+__all__ = ("PfsFluxReference",)
+
+
+class PfsFluxReference(pfs.datamodel.pfsFluxReference.PfsFluxReference):
+    """datamodels's ``PfsFluxReference`` with additional utility methods.
+    """
+    # No additional methods at present.
```

Comment:
If you don't need to extend the class, then you don't need this file. The reason we do it with other classes is because there are features we need for the pipeline that we don't make available for the general-purpose no-dependency datamodel package.

Reply:
I am deleting this file, and revising `PfsMapper.yaml` in `obs_pfs`, which has been pointing to `pfs.` *drp.stella* `.PfsFluxReference` class.